### DISCUSSION TITLE
PYROLYSIS OF DIFFERENT MATERIALS

### DISCUSSION THREAD

**Discussion by MarioCruzM** [firemodels/fds#13082] [Q&A]
Hi, I’m recently new to FDS, I’ve been reading the guides an everything but as always, there are so many questions unanswered. 
Let’s say I have a wooden pallet with cardboard boxes filled with broccoli. I have the wooden pallet data so I can insert it right away, but I haven’t found the chemical formulas to represent other complex materials, like cardboard boxes or veggies, etc. Does this material have such a tiny impact on pyrolysis that no data can be found (specially the cardboard box)? or is there another data base like SFPE Handbook that contains data for other complex materials? Thank you so much for your guidance and sorry if the question is too dumb.

**Comment by rmcdermo** [comment]
If you are new to FDS, or fire modeling in general, you need to be aware that there are two distinct types of modeling.  One is where you prescribe the mass loss rate of gaseous fuel for the flame (e.g., using HRRPUA with time RAMP in FDS).  This is the usual method for design fire scenarios in fire protection engineering, and is robust.  The other method is to try to predict the mass loss rate with a MATL reaction.  In this case, you are in research territory.

With that said, this paper has properties of cardboard:

M.B. McKinnon, S.I. Stoliarov, and A. Witkowski. Development of a pyrolysis model for corrugated cardboard. Combustion and Flame, 160:2595–2607, 2013.

Search the FDS Validation Guide on "cardboard" and you will see other examples you can follow.

I do not know if data on food.  But the "transient combustibles" series in the FDS Validation Guide may be a good place to start.

**Reply by MarioCruzM** [reply]
Thank you so much for your help. I'll take a look to the paper and those chapters.
